pinotdb/db.py

```diff
@@ -388,14 +388,43 @@ def finalize_query_payload(
             return {"sql": query}
 
     def normalize_query_response(self, input_query, query_response):
+        if query_response.status_code == 400:
```

Comment:
Make the condition more generic.

```suggestion
        if 400 <= query_response.status_code <  500:
```